Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.12-slim
+
+WORKDIR /app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY chatbot_app ./chatbot_app
+
+CMD ["uvicorn", "chatbot_app.main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
brakuje cache dla dependencies mozna zrobic tak ; 
COPY requirements.txt .
RUN pip install --no-cache-dir -r requirements.txt

Reply:
Mozna tez do enva dodac PYTHONDONTWRITEBYTECODE=1 i ENV PYTHONNUNBUFFERED=1 na poczatku i wtedy unikasz generowania .pyc i masz logi w czasie rzeczywistym. To sugestia jezeli uznasz ze zbedna to ok :D